run_conformance_tests.py

```diff
@@ -317,14 +316,13 @@ def main(argv=sys.argv[1:]):
 
     # Parse and gather all the script args
     (cmake_exe, build_system_name, build_system_call, full_conformance,
-     conformance_filter, implementation_name, additional_cmake_args, host_names,
+     implementation_name, additional_cmake_args, host_names,
```

Comment:
We use this script in internal testing, could we please keep the filter functionality in some form?

Reply:
Okay sure. Would using the exclude filter work for you though? It should be simple enough to switch from a whitelist to a blacklist, right?

Reply:
Great, then I will address this in my follow-up PR that fixes the `run_conformance_tests` script, if that's alright with you!

Reply:
That's fine, thank you